tests/manage/mcg/test_bucket_creation_deletion.py

```diff
@@ -148,14 +148,37 @@ class TestBucketCreation(MCGTest):
             "1-CLI-PVPOOL",
         ],
     )
-    def test_bucket_creation(
-        self, bucket_class_factory, bucket_factory, amount, interface, bucketclass_dict
+    def test_bucket_creation_deletion(
+        self,
+        verify_rgw_restart_count,
+        mcg_obj,
+        bucket_class_factory,
+        bucket_factory,
+        amount,
+        interface,
+        bucketclass_dict,
     ):
         """
         Test bucket creation using the S3 SDK, OC command or MCG CLI.
         The factory checks the bucket's health by default.
         """
-        bucket_factory(amount, interface, bucketclass=bucketclass_dict)
+        buckets = bucket_factory(amount, interface, bucketclass=bucketclass_dict)
+
+        if amount == 3 and interface == "OC":
```

Comment:
Please add an explicit clarification:
```suggestion
        # Checking BZ 2179271 on only one param
        if amount == 3 and interface == "OC":
```